buildscripts/buildvars-setup.sh

```diff
@@ -149,7 +149,7 @@ export __UnprocessedBuildArgs=
 export __CleanBuild=0
 export __VerboseBuild=0
 export __ClangMajorVersion=3
-export __ClangMinorVersion=5
+export __ClangMinorVersion=8
```

Comment:
Could you please also update https://github.com/dotnet/corert/blob/master/Documentation/prerequisites-for-building.md